src/pages/tools/audio/merge-audio/service.ts

```diff
@@ -0,0 +1,94 @@
+import { FFmpeg } from '@ffmpeg/ffmpeg';
+import { fetchFile } from '@ffmpeg/util';
+import { InitialValuesType } from './types';
+
+const ffmpeg = new FFmpeg();
+
+export async function mergeAudioFiles(
```

Comment:
this function only works if the input audio files have the same format. Try to combine aac and mp3